haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -1,9 +1,9 @@
 -- SPDX-FileCopyrightText: 2020 TQ Tezos
 -- SPDX-License-Identifier: MIT
 
--- | Tests for FA2 interface.
--- https://gitlab.com/tzip/tzip/-/blob/131b46dd89675bf030489ded9b0b3f5834b70eb6/proposals/tzip-12/tzip-12.md
-
+{- | Tests for FA2 interface.
+ https://gitlab.com/tzip/tzip/-/blob/131b46dd89675bf030489ded9b0b3f5834b70eb6/proposals/tzip-12/tzip-12.md
```

Comment:
This should be fixed to point at `tezos/tzip` (the repo was moved).

It also points at a very old commit, have you checked if we can (or perhaps even should) update to a newer one?

Reply:
I've updated all references to the repo to point to `tezos/tzip` in [5f1f4de](https://github.com/tqtezos/stablecoin/pull/187/commits/5f1f4deb90e7bb6b9d3caa6faa7c85a8461f94cb). However, I'm not so sure about updating commits.

Reply:
AFAICT, no `tzip` has changed between the commit we're referencing it at and `master` (which is not a huge surprise). So I can update, but I'm not sure what would be the point.

Reply:
AFAICT all related references in the repo (for the same tzip proposals) are roughly pointing at the same point in history and nothing major has since changed.

They may be updated (with some effort), but they are not so outdated to be actually wrong AFAICT, so let's keep them as-is.